CTFd/utils/uploads/uploaders.py

```diff
@@ -64,7 +64,8 @@ def download(self, filename):
 
     def delete(self, filename):
         if os.path.exists(os.path.join(self.base_path, filename)):
-            os.unlink(os.path.join(self.base_path, filename))
+            file_path = filename.split("/")[0]
```

Comment:
Shouldn't this be `os.path.split()`?

Reply:
Mildly concerned about what happens under Windows since the path seperator isn't `/`